include/revng/Yield/CrossRelations/RelationTarget.h

```diff
@@ -13,11 +13,12 @@ type: struct
 fields:
   - name: Kind
     type: RelationType
-  - name: Location
-    type: string
+  - name: Locations
+    sequence:
+      type: SortedVector
+      elementType: string
```

Comment:
Why do we have a set of locations?

Reply:
Before now the `cross-relations` used to look like
```yaml
Relations:
  - Location:        '/function/0x401040:Code_x86_64'
    Related:
      - Kind:            IsCalledFrom
        Location:        '/instruction/0x401100:Code_x86_64/0x401100:Code_x86_64/:Invalid'
      - Kind:            IsCalledFrom
        Location:        '/instruction/0x402100:Code_x86_64/0x402100:Code_x86_64/:Invalid'
      - Kind:            IsCalledFrom
        Location:        '/instruction/0x403100:Code_x86_64/0x403100:Code_x86_64/:Invalid'
  - Location:        '/dynamic-function/my_cool_function'
    Related:
      - Kind:            IsDynamicallyCalledFrom
        Location:        '/instruction/0x401149:Code_x86_64/0x401149:Code_x86_64/:Invalid'
```

Now it looks like
```yaml
Relations:
  - Location:        '/function/0x401040:Code_x86_64'
    Related:
      - Kind:            IsCalledFrom
        Locations:
          - '/instruction/0x401100:Code_x86_64/0x401100:Code_x86_64/:Invalid'
          - '/instruction/0x402100:Code_x86_64/0x402100:Code_x86_64/:Invalid'
          - '/instruction/0x403100:Code_x86_64/0x403100:Code_x86_64/:Invalid'
  - Location:        '/dynamic-function/my_cool_function'
    Related:
      - Kind:            IsCalledFrom
        Locations:       
          - '/instruction/0x401149:Code_x86_64/0x401149:Code_x86_64/:Invalid'
```

which is a lot easier to work with.